docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,58 @@
+Troubleshooting
+##############################################################################
+
+``csfg`` helper script
+==============================================================================
+
+I get a ``No such file or directory`` error when running the ``csfg`` script
+------------------------------------------------------------------------------
+
+Check your terminal working directory is within the ``cs-field-guide`` directory,
+the root directory of the project.
+Running the ``ls`` command in this directory should list the ``csfg`` file.
```

Comment:
What do I do if it is/is not there?

Reply:
Do you have any suggestions? Maybe if it's not there try re-cloning the repo or raising an issue on github?? Not sure

Reply:
I think I misinterpreted what was said here. I now think the idea is that if `ls` lists the `csfg` file then you'd be sure you were in the right directory. How about `if it's not there you may be in a different directory`

Reply:
@courtneycb thoughts?